packages/_infra/src/serve/db.ts

```diff
@@ -29,7 +29,7 @@ export const TileMetadataTableArn = {
     return cdk.Arn.format(
       {
         service: 'dynamodb',
-        region: '*',
```

Comment:
I dont think this will fix the problem, this was to allow other regions to write to dynamodb

Reply:
I see. I need to have a look what cause the problem. Just don't how to test this.

Reply:
where is it coming from, do you have the bundled index.js to see the stack trace?

Reply:
Yes, That's what I am thinking to get a bundle index.js first.

Reply:
![image](https://user-images.githubusercontent.com/12163920/168696505-0de2ae79-b473-4bcd-bbc1-6f37c705d1fc.png)
This the place of error, I can't find where is the place to using this. 
However, I am sure ti broke at the place for puting item to dynamodb.